GOVERNANCE.md

```diff
@@ -31,7 +31,7 @@ To become a Chaos Mesh member, you are expected to:
     - Join community discussions (e.g. community meetings, Slack).
 - Sponsored by at least 1 Chaos Mesh [maintainer or committer](https://github.com/chaos-mesh/chaos-mesh/blob/master/MAINTAINERS.md).
 
-Contributors that meet the above requirements will then be nominated by their sponsor at the next available [community meeting](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit) to become a member. The maintainers will confer and decide whether to grant membership or not. Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.
+Contributors that meet the above requirements will then be nominated by their sponsor through a PR. It is recommended to describe the reasons for the nomination and the contribution of the nominee in the PR. The maintainers will confer and decide whether to grant membership or not. Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.
```

Comment:
```suggestion
Contributors that meet the above requirements will then be nominated by their sponsor through a PR. It is recommended to describe the reasons for the nomination and the contribution of the nominee in the PR. To get the PR merged, at least two approvals are required from committers or maintainers. Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.
```

Reply:
OK. I will update the corresponding statements in the form below.